grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcService.java

```diff
@@ -280,9 +280,10 @@ private static void deframeAndRespond(
         final ArmeriaMessageDeframerHandler handler = new ArmeriaMessageDeframerHandler(
                 // Max outbound message size is handled by the GrpcService, so we don't need to set it here.
                 Integer.MAX_VALUE);
-        final HttpDeframer<DeframedMessage> deframer = HttpDeframer.of(handler, ctx.alloc());
-        StreamMessage.of(grpcResponse.content()).subscribe(deframer, ctx.eventLoop());
-        deframer.subscribe(singleSubscriber(unframedHeaders, res), ctx.eventLoop());
+        final StreamMessage<HttpData> publisher = StreamMessage.of(grpcResponse.content());
+        final StreamMessage<DeframedMessage> deframed =
+                new DefaultHttpDeframer<>(publisher, handler, ctx.alloc());
```

Comment:
Could we just build a fake `HttpResponse` here, so we don't have to use `DefaultHttpDeframer` directly? For example, you could just do `grpcResponse.toHttpResponse().decrame(handler, ctx.alloc())`.

Reply:
Fixed